src/charm.py

```diff
@@ -286,6 +287,37 @@ def _on_promote_user_admin_action(self, event: ActionEvent) -> None:
             return
         event.set_results(results)
 
+    def _on_anonymize_user_action(self, event: ActionEvent) -> None:
+        """Promote user admin and report action result.
```

Comment:
The description seems off, it speaks about promoting an user rather than anonymizing.